install_config/syncing_groups_with_ldap.adoc

```diff
@@ -433,6 +433,153 @@ users:
+type indicated a mis-configured user query. However, if it is certain that the user
+query is properly configured, these errors can be tolerated and the out of scope members
+ignored.
+
+Similarly, when the group sync process fails to locate a member for a group, it fails
+outright. Failures of this type most often indicate mis-configured LDAP group configurations
+on the LDAP server. However, if it is certain that the group entries on the LDAP server
+have been configured correctly, these errors can be tolerated and the problematic
+members ignored.
+
+[NOTE]
+====
+Enabling error tolerances for the LDAP group sync will cause the sync process to ignore
+problematic member entries. If the LDAP group sync is not configured correctly, this will
+result in synced OpenShift groups missing members.
+====
+
+.LDAP Entries Using RFC 2307 Schema With Problematic Group Membership: *_rfc2307_problematic_users.ldif_*
+====
+[source,ldif]
+----
+  dn: ou=users,dc=example,dc=com
+  objectClass: organizationalUnit
+  ou: users
+
+  dn: cn=Jane,ou=users,dc=example,dc=com
+  objectClass: person
+  objectClass: organizationalPerson
+  objectClass: inetOrgPerson
+  cn: Jane
+  sn: Smith
+  displayName: Jane Smith
+  mail: jane.smith@example.com
+
+  dn: cn=Jim,ou=users,dc=example,dc=com
+  objectClass: person
+  objectClass: organizationalPerson
+  objectClass: inetOrgPerson
+  cn: Jim
+  sn: Adams
+  displayName: Jim Adams
+  mail: jim.adams@example.com
+
+  dn: ou=groups,dc=example,dc=com
+  objectClass: organizationalUnit
+  ou: groups
+
+  dn: cn=admins,ou=groups,dc=example,dc=com
+  objectClass: groupOfNames
+  cn: admins
+  owner: cn=admin,dc=example,dc=com
+  description: System Administrators
+  member: cn=Jane,ou=users,dc=example,dc=com
+  member: cn=Jim,ou=users,dc=example,dc=com
+  member: cn=INVALID,ou=users,dc=example,dc=com <1>
+  member: cn=Jim,ou=OUTOFSCOPE,dc=example,dc=com <2>
+----
+<1> This member doesn't exist on the LDAP server.
+<2> This member may exist, but is not under the base DN in the
+user query for the sync job.
+====
+
+In order to tolerate errors, the following additions to your sync configuration
+file must be made:
+
+.LDAP Sync Configuration Using RFC 2307 Schema Tolerating Errors: *_rfc2307_config_tolerating.yaml_*
+====
+[source,yaml]
+----
+kind: LDAPSyncConfig
+apiVersion: v1
+url: ldap://LDAP_SERVICE_IP:389
+insecure: true
+rfc2307:
+    groupsQuery:
+        baseDN: "ou=groups,dc=example,dc=com"
+        scope: sub
+        derefAliases: never
+        filter: (objectclass=groupOfNames)
+    groupUIDAttribute: dn
+    groupNameAttributes: [ cn ]
+    groupMembershipAttributes: [ member ]
+    usersQuery:
+        baseDN: "ou=users,dc=example,dc=com"
+        scope: sub
+        derefAliases: never
+        filter: (objectclass=inetOrgPerson)
+    userUIDAttribute: dn
+    userNameAttributes: [ mail ]
+    tolerateMemberNotFoundErrors: true <1>
+    tolerateMemberOutOfScopeErrors: true <2>
```

Comment:
include the config fields, set to `false` in all the rfc sample configs so when they copy/paste them, they'll have them in their configs